modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc

```diff
@@ -1,70 +1,182 @@
+This section provides the requirements and recommendations for deploying Redpanda.
 
-* Mount an XFS or ext4 file system on any storage volumes that host the data directory of Redpanda (`/var/lib/redpanda/data`) or the Tiered Storage cache. XFS is highly recommended. NFS is not supported.
-* Use locally-attached NVMe devices. RAID-0 is required if you use multiple disks.
-* Use ephemeral cloud instance storage only in combination with Tiered Storage or for Tiered Storage cache. Without Tiered Storage, attached persistent volumes (for example, EBS).
+=== Tuning
 
-To learn what volumes Redpanda recommends, see the xref:./kubernetes-best-practices.adoc#storage[storage best practices].
-To learn how to configure storage, see xref:manage:kubernetes/configure-storage.adoc[Configure storage].
+*Requirement*: Before deploying Redpanda to production, each worker node that runs Redpanda must be tuned to optimize the Linux kernel for Redpanda processes. See xref:./kubernetes-tune-workers.adoc[Tuning Kubernetes Worker Nodes for Production].
 
-== Object storage providers for Tiered Storage
+=== Deploy at least three Pod replicas
 
-* Amazon Simple Storage Service (S3)
-* Google Cloud Storage (GCS), using the Google Cloud Platform S3 API
-* Azure Blob Storage (ABS)
+*Recommendation*: Redpanda Data recommends at least three Pod replicas (Redpanda brokers) to use as _seed servers_. Seed servers are used to bootstrap the gossip process for new brokers joining a cluster. When a new broker joins, it connects to the seed servers to find out the topology of the Redpanda cluster. A larger number of seed servers makes consensus more robust and minimizes the chance of unwanted clusters forming when brokers are restarted without any data.
 
-== External networking
+NOTE: By default, the Redpanda Helm chart deploys a StatefulSet with three Redpanda brokers. You can specify the number of Redpanda brokers in the xref:reference:redpanda-helm-spec.adoc#statefulset-replicas[`statefulset.replicas`] configuration.
 
-For external access, each worker node in your cluster must have a static, externally accessible IP address to allow clients to connect to the NodePort Service and access the Redpanda broker.
-Redpanda uses the following default ports:
+=== Set resource requests and limits for memory and CPU
+
+*Recommendation*: In a production cluster, the resources you allocate to Redpanda should be proportionate to your machine type. Redpanda Data recommends that you determine and set these values before deploying the cluster, but you can also update the values on a running cluster. For instructions on setting Pod resources, see xref:manage:kubernetes/manage-resources.adoc[Manage Pod Resources in Kubernetes].
+
+=== Use local PersistentVolumes backed by NVMe disks
+
+*Recommendation*: Use PersistentVolumes that are backed by locally attached NVMe devices to store the Redpanda data directory. NVMe devices outperform traditional SSDs or HDDs.
```

Comment:
*Requirement*: For external access, each worker node in your cluster must have a static, externally-accessible IP address.